src/common/pubsub.ts

```diff
@@ -47,7 +47,6 @@ const userDeletedTopic = pubsub.topic('user-deleted');
 const userUpdatedTopic = pubsub.topic('user-updated');
 const usernameChangedTopic = pubsub.topic('username-changed');
 const settingsUpdatedTopic = pubsub.topic('settings-updated');
-const notificationsReadTopic = pubsub.topic('api.v1.notifications-read');
```

Comment:
You also removed this from streams?

Reply:
Good point! 🙈

Reply:
https://github.com/dailydotdev/streams/pull/24